src/main/java/thePackmaster/cards/pinnaclepack/MysteryCroquettesSpecialColourless.java

```diff
@@ -0,0 +1,30 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.VulnerablePower;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class MysteryCroquettesSpecialColourless extends AbstractPinnacleCard {
+
+    public final static String ID = makeID("MysteryCroquettesSpecialColourless");
+    private static final int MAGIC = 1;
+    private static final int UPGRADE_MAGIC = 99;
+
+    public MysteryCroquettesSpecialColourless() {
```

Comment:
*Mystery Croquette*
This is fine for a token card but the upgrade is out of line and needs to change. I'd recommend making this same values as the one that gives weak.